mathics/builtin/arithmetic.py

```diff
@@ -1861,6 +1861,23 @@ class Factorial(PostfixOperator, _MPMathFunction):
     mpmath_name = "factorial"
 
 
+class Factorial2(PostfixOperator, _MPMathFunction):
+    """
+    <dl>
+    <dt>'Factorial2[$n$]'
+    <dt>'$n$!!'
```

Comment:
BTW I prefer indenting the `<dt>`s inside the `<dl>` like the way other XML tags work.   Yeah, I don't know why we don't add the ends to some things like `</dt>` or `</dd>` or `</li>`. 

It would be nice also to say _what_ Factorial2 is.

Reply:
> BTW I prefer indenting the <dt>s inside the <dl> like the way other XML tags work. 

I too, I just made it because I saw this indentation elsewhere.

> Yeah, I don't know why we don't add the ends to some things like </dt> or </dd> or </li>.

Adding them work too.

> It would be nice also to say what Factorial2 is.

Maybe `<dd>computes the double factorial of $n$.`?